test/e2e/e2e.go

```diff
@@ -102,6 +103,26 @@ func (e *efsDriver) GetPersistentVolumeSource(readOnly bool, fsType string, volu
 	return &pvSource, nil
 }
 
+func (e *efsDriver) GetDynamicProvisionStorageClass(config *testsuites.PerTestConfig, fsType string) *storagev1.StorageClass {
```

Comment:
Is implementing `GetDynamicProvisionStorageClass` how to enable upstream dynamic provision tests?

Reply:
Yeah, this allows the test suites to detect and run dynamic provision tests